pkg/proxy/util/utils.go

```diff
@@ -254,7 +254,7 @@ func GetNodeAddresses(cidrs []string, nw NetworkInterfacer) (sets.String, error)
 // LogAndEmitIncorrectIPVersionEvent logs and emits incorrect IP version event.
 func LogAndEmitIncorrectIPVersionEvent(recorder events.EventRecorder, fieldName, fieldValue, svcNamespace, svcName string, svcUID types.UID) {
 	errMsg := fmt.Sprintf("%s in %s has incorrect IP version", fieldValue, fieldName)
-	klog.Errorf("%s (service %s/%s).", errMsg, svcNamespace, svcName)
+	klog.ErrorS(nil, "Incorrect IP version", "service", klog.KRef(svcName, svcNamespace), "field", fieldName, "value", fieldValue)
```

Comment:
klog.KRef(podNamespace, podName)

Reply:
@thockin suggested it should be `klog.KRef(svcName, svcNamespace)` and if we see, we have to log service name and namespace and not pod name and namespace. I might be wrong though. // @yangjunmyfm192085

Reply:
> @thockin suggested it should be `klog.KRef(svcName, svcNamespace)` and if we see, we have to log service name and namespace and not pod name and namespace. I might be wrong though. // @yangjunmyfm192085

Sorry， my mistake, my mean, it's  `klog.KRef(svcNamespace,svcName)`

Reply:
You can change to
`klog.ErrorS(nil, "Incorrect IP version", "service", klog.KRef(svcNamespace, svcName,), "field", fieldName, "value", fieldValue)`

Reply:
Done @yangjunmyfm192085 🚀

Reply:
thanks